app/src/main/java/com/cornellappdev/uplift/ui/screens/ProfileCreationScreen.kt

```diff
@@ -0,0 +1,227 @@
+                        fontSize = 12.sp,
+                        fontFamily = montserratFamily,
+                        fontWeight = FontWeight.Normal
+                    )
+                }
+
+                Row(verticalAlignment = Alignment.CenterVertically) {
+                    Checkbox(
+                        checked = checkedState3.value,
+                        onCheckedChange = { checkedState3.value = it },
+                        colors = checkboxColors
+                    )
+                    Text(
+                        text = "I allow Uplift to access my location",
+                        fontSize = 12.sp,
+                        fontFamily = montserratFamily,
+                        fontWeight = FontWeight.Normal
+                    )
+                }
+
+            }
+
+            if (checkedState.value && checkedState2.value && checkedState3.value) {
+                Spacer(modifier = Modifier.height(155.dp))
+
+                Row(
+                    verticalAlignment = Alignment.CenterVertically,
+                    modifier = Modifier.padding(bottom = 25.dp)
+                ) {
+                    Text(
+                        text = "Are you ready to",
+                        fontSize = 16.sp,
+                        fontFamily = montserratFamily,
+                        fontWeight = FontWeight.Bold
+                    )
+
+                    Spacer(modifier = Modifier.width(6.dp))
+
+                    Image(
+                        painter = painterResource(R.drawable.ic_main_logo),
+                        contentDescription = "",
+                        modifier = Modifier.size(width = 26.dp, height = 23.dp)
+                    )
+
+                    Image(
+                        painter = painterResource(R.drawable.lift_logo),
+                        contentDescription = "",
+                    )
+                    Spacer(modifier = Modifier.width(6.dp))
+
+                    Text(
+                        text = "?",
+                        fontSize = 16.sp,
+                        fontFamily = montserratFamily,
+                        fontWeight = FontWeight.Bold
+                    )
+                }
+            } else {
+                Spacer(modifier = Modifier.height(205.dp))
+            }
+            Button(
+                {},
+                enabled = checkedState.value && checkedState2.value && checkedState3.value,
+                colors = ButtonDefaults.buttonColors(
+                    backgroundColor = PRIMARY_YELLOW,
+                    disabledBackgroundColor = GRAY02,
+                    contentColor = Color.Black
+                ),
+                shape = RoundedCornerShape(38.dp),
+                modifier = Modifier.size(height = 44.dp, width = 144.dp)
+            ) {
+                Text(
+                    text = if (checkedState.value && checkedState2.value && checkedState3.value) "Get started" else "Next",
```

Comment:
...and replace all other occurrences of this `&&` computation with `allChecked`.